tests/data_extraction/test_metadata_extraction_emodul.py

```diff
@@ -7,7 +7,7 @@ def test_metadata_extraction_emodul():
     """Tesing the metadata extraction on a single example"""
 
     # setting up the test example
-    input = '../usecases/MinimumWorkingExample/Data/E-modul/BA-Losert MI E-Modul 28d v. 04.08.14 Probe 4'
+    input = '../../usecases/MinimumWorkingExample/Data/E-modul/BA-Losert MI E-Modul 28d v. 04.08.14 Probe 4'
```

Comment:
This change needs to be reversed.
As specified within: [lebedigital.yml](https://github.com/BAMresearch/LebeDigital/blob/main/.github/workflows/lebedigital.yml), the actions run pytest within the tests folder. If you look at the main branch you can see it working.